go.mod

```diff
@@ -2,6 +2,12 @@ module github.com/pomerium/pomerium
 
 go 1.25.1
 
+// breaking change in go 1.25.0 changed the behaviour of x509.CreateCertificate:
+// https://go-review.googlesource.com/c/go/+/674477
```

Comment:
Informal proposal : https://github.com/golang/go/issues/71746

This will break the test in https://github.com/pomerium/pomerium/blob/efb100b29e46c7bf8a19acb559034eb03bf59bf3/config/config_test.go#L16, but more generally is a (probably good) change in default behavior in https://pkg.go.dev/crypto/x509#CreateCertificate 

I'm not really sure of the wider implications of this change in pomerium...